chain/types/ethtypes/eth_types.go

```diff
@@ -642,7 +643,7 @@ func (e *EthAddressList) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
-// TopicSpec represents a specification for matching by topic. An empty spec means all topics
+// EthTopicSpec represents a specification for matching by topic. An empty spec means all topics
```

Comment:
I have a lot of little cleanups here. Can we break them out into separate pull requests? Maybe a few pull requests.